templates/webapps/reports/jobs_errors_per_tool.mako

```diff
@@ -21,7 +21,7 @@ ${get_css()}
 
 <!--jobs_errors_per_tool.mako-->
 <div class="toolForm">
-    <div class="toolFormBody">
+    <div class="reportToolFormBody">
```

Comment:
can we please rename this to something like 'reportBody'; this has nothing to do with the toolForm and now is a good time :)